farmdata2_modules/fd2_tabs/fd2_field_kit/fd2_field_kit.info

```diff
@@ -3,4 +3,5 @@ description = A collection of forms for fast and accurate data entry in the fiel
 core = 7.x
 package = FarmData2
 
-scripts[] = '../resources/FarmData2.js'
\ No newline at end of file
+scripts[] = '../resources/FarmData2.js'
+scripts[] = '../resources/RegexInputComponent.js'
```

Comment:
Let's remove all changes to the FieldKit for this PR.  We'll update the FieldKit to include this component in the PR for #285.